CMakeLists.txt

```
@@ -9,11 +9,28 @@
 project(FreeRCT)
 cmake_minimum_required(VERSION 2.8)
 
+# Add a compiler flag VAR_NAME=FLAG. If VAR_NAME is already set, FLAG is appended.
+FUNCTION(add_flags VAR_NAME FLAG)
+	IF(${VAR_NAME})
```

```suggestion
	IF(DEFINED VAR_NAME)
```
Isn't this more proper? I'm unsure if I understand this right.

---

Good catch, fixed. However VAR_NAME is a variable containing the name of the variable to modify, so we still need to dereference it in order to check if the variable exists.